webui/common/src/utilities/misc/renderFileNameLink.tsx

```diff
@@ -1,3 +1,14 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
 import {Link} from 'react-router-dom';
 import React from 'react';
```

Comment:
Is there an import order we should follow? Would be nice to get all these style things in a linter, especially since we are committing so much code (should be done in a separate PR)